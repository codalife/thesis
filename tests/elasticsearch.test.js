const script = require('../dbs/elasticsearch');

test('has have methods', () => {
	expect(script).toMatchSnapshot();
})