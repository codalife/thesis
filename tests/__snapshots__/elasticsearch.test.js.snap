// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has have methods 1`] = `
Object {
  "addToIndex": [Function],
  "closeConnection": [Function],
  "createIndex": [Function],
  "dropIndex": [Function],
  "getFromIndex": [Function],
  "search": [Function],
  "waitForIndexing": [Function],
}
`;
